opendevin/runtime/docker/ssh_box.py

```diff
@@ -344,21 +412,25 @@ def _send_interrupt(
             f'Command: "{cmd}" timed out. Sending SIGINT to the process: {command_output}',
         )
 
-    def execute(self, cmd: str, timeout: int | None = None) -> tuple[int, str]:
+    def execute(
+        self, cmd: str, stream: bool = False, timeout: int | None = None
```

Comment:
Why stream is disabled by default?

Reply:
I think streaming output makes sense for initialization or time-consuming operations to get feedback. Currently, we don't support streaming output to the frontend. And the agent prefers not to receive the streaming output.

Reply:
How about only displaying the output to the logs?